lib/sensors/tof.cpp

```diff
@@ -1,4 +1,4 @@
-#include "time_of_flight.hpp"
+#include "tof.hpp"
```

Comment:
[Why was this changed back?](https://github.com/Hyp-ed/hyped-2024/pull/25#pullrequestreview-1723125739)